src/pages/ServiceDetails/ServiceInfo/ServiceInfoRouteRules/RouteRuleRoute.tsx

```diff
@@ -74,6 +83,22 @@ class RouteRuleRoute extends React.Component<RouteRuleRouteProps> {
     }));
   }
 
+  bulletChartValues = () => {
+    return (this.props.route || []).map((routeItem, u) => ({
+      value: this.props.route.length === 1 ? 100 : routeItem.weight,
+      title: `${u}_${routeItem.weight}`,
+      color: PFBlueColors[u],
```

Comment:
Since we have a ranged set of colors, this line would break in case we have more than 6 versions. I would protect that somehow. For example, repeating colors:

```
color: PFBlueColors[u%PFBlueColors.length]
```
using that, we will only use values from 0 to 5.